VAMobile/src/utils/filesystem.ts

```diff
@@ -191,11 +192,11 @@ export const arrayIncludesArray = (variableArray: number[], fixedArray: number[]
  * within the file content. The "/Encrypt" signature is also added to view-only
  * PDFs as well as password-encrypted PDF files
  * Logic is based on vets-website file utility function checkIsEncryptedPdf
- * @param uri - Uri string of the file
+ * @param document - DocumentPickerResponse file document
  */
-export const isPdfEncrypted = async (uri: string): Promise<boolean> => {
-  const base64String = await getBase64ForUri(uri)
-  if (!base64String) {
+export const isPdfEncrypted = async (document: DocumentPickerResponse): Promise<boolean> => {
+  const base64String = await getBase64ForUri(document.uri)
+  if (!base64String || !document.name.toLowerCase().endsWith('pdf')) {
```

Comment:
Can we use the type or another property on the document rather than parsing the name?